src/backend/utils.ts

```diff
@@ -1306,9 +1306,16 @@ export const writeConfig = (appName: string, config: Partial<AppSettings>) => {
   }
 }
 
+const COPY_TIMEOUT_MS = 30000 // wait time before throwing a timeout error
```

Comment:
we won't exceed this during normal operation on slow machines will we?

Reply:
I dont think so, that is actually a pretty generous time limit just to avoid being stuck forever. the file would need to be huge to exceed this imo.